Formula/ffmpeg.rb

```diff
@@ -4,7 +4,7 @@ class Ffmpeg < Formula
   url "https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.xz"
   version "4.3.1-with-options" # to distinguish from homebrew-core's ffmpeg
   sha256 "ad009240d46e307b4e03a213a0f49c11b650e445b1f8be0dda2a9212b34d2ffb"
-  revision 2
+  revision 3
```

Comment:
You could possibly also add
```rb
license "GPL-2.0-or-later"
```
just before.

Reply:
good one, actually just did it on homebrew-core too ^^
https://github.com/Homebrew/homebrew-core/pull/60410#issuecomment-683584725

Reply:
perfect, thanks a lot!